core/tests/performance_tests/test_config.py

```diff
@@ -31,18 +31,108 @@
         cached: see above.
 """
 
-SPLASH_PAGE_KEY = 'splash'
+PAGE_KEY_SPLASH = 'splash'
+PAGE_KEY_LIBRARY = 'library'
+PAGE_KEY_CREATOR_DASHBOARD = 'dashboard'
+PAGE_KEY_EXPLORATION_PLAYER = 'explore'
+PAGE_KEY_EXPLORATION_EDITOR = 'editor'
+PAGE_KEY_COLLECTION_PLAYER = 'collection'
+PAGE_KEY_PROFILE_PAGE = 'profile'
+
+# Default port for GAE server that is started for performance tests.
+PERFORMANCE_TESTS_SERVER_PORT = 9501
+
+# Possible preload options. Only either one of them could be used and they are
```

Comment:
I suggest: "Possible preload options. Exactly one of these can be used."

and defining PRELOAD_NONE = 'NONE' and checking explicitly for that.